<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QDialog" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>516</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>903</width>
    <height>516</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>903</width>
    <height>516</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/assets/images/editor/editor.png);</string>
  </property>
  <widget class="QPushButton" name="musicButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 13pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);
background-image: transparent;
border-image: transparent;</string>
   </property>
   <property name="text">
    <string>Music</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);
background-image: transparent;
border-image: transparent;</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);
background-image: transparent;
border-image: transparent;</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="item_1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BusyCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
border-image: url(:/assets/images/editor/box.png);
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cleanButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);
background-image: transparent;
border-image: transparent;</string>
   </property>
   <property name="text">
    <string>Clean</string>
   </property>
  </widget>
  <widget class="QPushButton" name="item_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>128</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BusyCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
border-image: url(:/assets/images/editor/horizontal_wood.png);
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="item_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BusyCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
border-image: url(:/assets/images/editor/one_block.png);
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="item_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>32</width>
     <height>128</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BusyCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
border-image: url(:/assets/images/editor/vertical_wood.png);
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
